src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ListingHelper.java

```diff
@@ -144,6 +152,10 @@ public static ListingType getListingTypeFromCaseData(ListingData listingData, Ca
                     ? DocumentHelper.getHearingDuration(hearingType)
                     : " ");
 
+            log.info("checkScotlandReadingDeliberationDay");
```

Comment:
this log message (and some below as well) can be removed (may be later)

Reply:
Removed